frontend/lib/laletterbuilder/letter-builder/habitability/letter-email-to-user.tsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+
+import { asEmailStaticPage } from "../../../static-page/email-static-page";
+import { li18n } from "../../../i18n-lingui";
+import { t, Trans } from "@lingui/macro";
+import { USPS_TRACKING_URL_PREFIX } from "../../../../../common-data/loc";
+import { HtmlEmail } from "../../../static-page/html-email";
+import { TransformSession } from "../../../util/transform-session";
+import { AllSessionInfo } from "../../../queries/AllSessionInfo";
+
+type EmailBodyProps = {
+  firstName: string;
+  trackingNumber?: string;
+  isInLosAngeles: boolean;
+};
+
+const EmailBody: React.FC<EmailBodyProps> = (props) => (
+  <>
+    <Trans>
+      <p>Hello {props.firstName},</p>
+      <p>
+        You've sent your Notice To Repair letter. Attached to this email is a
+        PDF copy for your records.
+      </p>
+    </Trans>
+    {props.trackingNumber && (
+      <p>
+        <Trans>
+          You can also track the delivery of your letter using USPS Tracking:
+        </Trans>{" "}
+        <a
+          data-jf-show-href-only-in-plaintext
+          href={USPS_TRACKING_URL_PREFIX + props.trackingNumber}
+        >
+          {props.trackingNumber}
+        </a>
+      </p>
+    )}
+    {props.isInLosAngeles ? (
```

Comment:
also for my own clarification, is there any concern about this excluding users from unincorporated areas which SAJE might still serve?